docs/Linuxbrew.md

```diff
@@ -61,9 +61,9 @@ sudo apt-get install build-essential curl file git
 sudo yum groupinstall 'Development Tools' && sudo yum install curl file git
 ```
 
-### Raspberry Pi
+### ARM
 
-Linuxbrew can run on Raspberry Pi (32-bit ARM), but no binary packages (bottles) are available. Support for Raspberry Pi is on a best-effort basis. Pull requests are welcome to improve the experience on Raspberry Pi.
+Linuxbrew can run on 32-bit ARM (Raspberry Pi) and 64-bit ARM (AArch64), but no binary packages (bottles) are available. Support for ARM is on a best-effort basis. Pull requests are welcome to improve the experience on ARM and Raspberry Pi.
```

Comment:
```suggestion
Linuxbrew can run on 32-bit ARM (Raspberry Pi and others) and 64-bit ARM (AArch64), but no binary packages (bottles) are available. Support for ARM is on a best-effort basis. Pull requests are welcome to improve the experience on ARM platforms.
```

Reply:
Would it be worth mentioning the portable ruby packages etc.? Since those help people bootstrap the linuxbrew ARM experience, and they don't have to start from zero.

Also, feel free to dismiss my review if you completely disagree.